Building on Windows machine fails for sqlite3-parser

To learn using Rust with a focus on web and SQLite, we have created a working solution based on actix-web and turso using this library.
The project was created on a macOS machine and is running fine there.
Me, building it on a Windows machine fails and really have no clue how to solve the issue.

I get the following error:

error: failed to run custom build command for `sqlite3-parser v0.8.0`

Caused by:
  process didn't exit successfully: `C:\Users\gokkep\projects\rust\tursotest\target\debug\build\sqlite3-parser-9f9fcbb7d7225a5f\build-script-build` (exit code: 101)
  --- stdout
  OPT_LEVEL = Some("1")
  HOST = Some("x86_64-pc-windows-msvc")
  cargo:rerun-if-env-changed=CC_x86_64-pc-windows-msvc
  CC_x86_64-pc-windows-msvc = None
  cargo:rerun-if-env-changed=CC_x86_64_pc_windows_msvc
  CC_x86_64_pc_windows_msvc = None
  cargo:rerun-if-env-changed=HOST_CC
  HOST_CC = None
  cargo:rerun-if-env-changed=CC
  CC = None
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  DEBUG = Some("true")
  cargo:rerun-if-env-changed=CFLAGS_x86_64-pc-windows-msvc
  CFLAGS_x86_64-pc-windows-msvc = None
  cargo:rerun-if-env-changed=CFLAGS_x86_64_pc_windows_msvc
  CFLAGS_x86_64_pc_windows_msvc = None
  cargo:rerun-if-env-changed=HOST_CFLAGS
  HOST_CFLAGS = None
  cargo:rerun-if-env-changed=CFLAGS
  CFLAGS = None

I attached the used toml file and lock file 

[Cargo.zip](https://github.com/libsql/libsql-client-rs/files/12431282/Cargo.zip)

I pursued this issue myself. I did found that the cargo.toml file of this library has to depend on at least version 0.9.0 of the sqlite3-parser rust library. I noticed @samishoux did fork this library and had made that particular change.

So I added the following line to the cargo.toml file of my project:
`libsql-client = { git = "https://github.com/samishoux/libsql-client-rs" }`

After that my project could build, but could not link, with the following error: 
LINK : fatal error LNK1181: cannot open input file 'sqlite3.lib'

To solve this latest issue, I installed the sqlite3 package through vcpkg, with the following command inside my vcpkg environment:
`.\vcpkg.exe install --triplet=x64-windows-static-md sqlite3`, this installs package sqlite3 for Windows x64 environment and will be statically linked against the generated binary executable.

I saw that this library has an outstanding [PR](https://github.com/libsql/libsql-client-rs/pull/32), which also updates the dependency of sqlite3-parser to a version that will build on Windows.

Could someone please update this, so I can use this repo again?

For me this issue can be closed.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -15,23 +15,34 @@ base64 = "0.21.0"
 num-traits = "0.2.15"
 serde_json = "1.0.91"
 worker = { version = "0.0.18", optional = true }
-spin-sdk = { version = "2.0", git = "https://github.com/fermyon/spin", tag = "v2.0.0", default-features = false, features = ["http"], optional = true }
-sqlite3-parser = { version = "0.8.0", default-features = false, features = [ "YYNOERRORRECOVERY" ] }
+spin-sdk = { version = "2.0", git = "https://github.com/fermyon/spin", tag = "v2.0.0", default-features = false, features = [
+    "http",
+], optional = true }
+sqlite3-parser = { version = "0.12.0", default-features = false, features = [
+    "YYNOERRORRECOVERY",
+] }
 http = { version = "0.2", optional = true }
 bytes = { version = "1.4.0", optional = true }
 anyhow = "1.0.69"
-reqwest = { version = "0.11.14", optional = true, default-features = false, features = ["rustls-tls"] }
+reqwest = { version = "0.11.14", optional = true, default-features = false, features = [
+    "rustls-tls",
+] }
 hrana-client = { version = "0.3", optional = true }
 hrana-client-proto = { version = "0.2" }
 futures-util = { version = "0.3.21", optional = true }
 serde = "1.0.159"
 tracing = "0.1.37"
 futures = "0.3.28"
-fallible-iterator = "0.2.0"
+fallible-iterator = "0.3.0"
 libsql = { version = "=0.1.8", optional = true }
 
 [features]
-default = ["local_backend", "hrana_backend", "reqwest_backend", "mapping_names_to_values_in_rows"]
+default = [
+    "local_backend",
+    "hrana_backend",
+    "reqwest_backend",
+    "mapping_names_to_values_in_rows",
+]
 workers_backend = ["worker", "futures-util"]
 reqwest_backend = ["reqwest"]
 local_backend = ["libsql"]

